x-pack/plugins/osquery/public/live_queries/form/packs_combobox_field.tsx

```diff
@@ -36,81 +34,66 @@ interface PackOption {
   description?: string;
 }
 
-export const PacksComboBoxField = ({ field, euiFieldProps = {}, idAria, ...rest }: Props) => {
+export const PacksComboBoxField = ({
```

Comment:
can we move `useController` inside this component?

Reply:
done